src/test/java/com/conveyal/gtfs/loader/GtfsFlexTest.java

```diff
@@ -0,0 +1,82 @@
+package com.conveyal.gtfs.loader;
+
+import com.conveyal.gtfs.TestUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+
+import static com.conveyal.gtfs.GTFS.load;
+import static com.conveyal.gtfs.GTFS.validate;
+import static com.conveyal.gtfs.TestUtils.assertThatSqlCountQueryYieldsExpectedCount;
+
+/**
+ * Load in a GTFS feed with GTFS flex features, and ensure all needed fields are imported correctly.
+ * TODO: update feed to use more features, and test for these.
+ */
+public class GtfsFlexTest {
+    private static String testDBName;
+    private static DataSource testDataSource;
+    private static String testNamespace;
+
+    @BeforeAll
+    public static void setUpClass() throws IOException {
+        // Create a new database
+        testDBName = TestUtils.generateNewDB();
+        String dbConnectionUrl = String.format("jdbc:postgresql://localhost/%s", testDBName);
+        testDataSource = TestUtils.createTestDataSource(dbConnectionUrl);
+        // load feed into db
+        String zipFileName = TestUtils.zipFolderFiles(
+            "real-world-gtfs-feeds/washington-park-shuttle-with-flex-additions",
+            true);
+        FeedLoadResult feedLoadResult = load(zipFileName, testDataSource);
+        testNamespace = feedLoadResult.uniqueIdentifier;
+        validate(testNamespace, testDataSource);
+    }
+
+    @Test
+    void canImportContinuousPickupAndDropoff() {
```

Comment:
Change this to a parameterized test. See https://github.com/conveyal/gtfs-lib/blob/dev/src/test/java/com/conveyal/gtfs/loader/ConditionallyRequiredTest.java

Reply:
Excellent suggestion thank you! Refactored in d715be09dfdcf79dbd2e565cbf958359e34eec6b